client/live_test.go

```diff
@@ -2,8 +2,8 @@ package client_test
 
 import (
 	gc "gopkg.in/check.v1"
-
```

Comment:
This blank line is needed back in.

As a general rule, we separate imports into three different blocks: system packages, other package, packages from this repo.